runtime/mozart/cli/src/command.rs

```diff
@@ -0,0 +1,485 @@
+    fn impl_name() -> String {
+        "Tanssi".into()
+    }
+
+    fn impl_version() -> String {
+        let commit_hash = env!("SUBSTRATE_CLI_COMMIT_HASH");
+        format!("{}-{commit_hash}", NODE_VERSION)
+    }
+
+    fn description() -> String {
+        env!("CARGO_PKG_DESCRIPTION").into()
+    }
+
+    fn author() -> String {
+        env!("CARGO_PKG_AUTHORS").into()
+    }
+
+    fn support_url() -> String {
+        "https://github.com/moondance-labs/tanssi/issues/new".into()
+    }
+
+    fn copyright_start_year() -> i32 {
+        2017
+    }
+
+    fn executable_name() -> String {
+        "tanssi".into()
+    }
+
+    fn load_spec(&self, id: &str) -> std::result::Result<Box<dyn sc_service::ChainSpec>, String> {
+        let id = if id.is_empty() {
+            let n = get_exec_name().unwrap_or_default();
+            ["mozart"]
+                .iter()
+                .cloned()
+                .find(|&chain| n.starts_with(chain))
+                .unwrap_or("mozart")
+        } else {
+            id
+        };
+        Ok(match id {
+            #[cfg(feature = "mozart-native")]
+            "mozart" => Box::new(tanssi_relay_service::chain_spec::mozart_config()?),
+            #[cfg(feature = "mozart-native")]
+            "dev" | "mozart-dev" => {
+                Box::new(tanssi_relay_service::chain_spec::mozart_development_config()?)
+            }
+            #[cfg(feature = "mozart-native")]
+            "mozart-local" => {
+                Box::new(tanssi_relay_service::chain_spec::mozart_local_testnet_config()?)
+            }
+            #[cfg(feature = "mozart-native")]
+            "mozart-staging" => {
+                Box::new(tanssi_relay_service::chain_spec::mozart_staging_testnet_config()?)
+            }
+            #[cfg(not(feature = "mozart-native"))]
+            name if name.starts_with("mozart-") && !name.ends_with(".json") || name == "dev" => {
+                Err(format!(
+                    "`{}` only supported with `mozart-native` feature enabled.",
+                    name
+                ))?
+            }
+            path => {
+                let path = std::path::PathBuf::from(path);
+
+                let chain_spec = Box::new(polkadot_service::GenericChainSpec::from_json_file(
+                    path.clone(),
+                )?) as Box<dyn polkadot_service::ChainSpec>;
+
+                // When `force_*` is given or the file name starts with the name of one of the known
+                // chains, we use the chain spec for the specific chain.
+                if self.run.force_mozart {
+                    Box::new(
+                        tanssi_relay_service::chain_spec::MozartChainSpec::from_json_file(path)?,
+                    )
+                } else {
+                    chain_spec
+                }
+            }
+        })
+    }
+}
+
+#[allow(clippy::borrowed_box)]
+fn set_default_ss58_version(_spec: &Box<dyn polkadot_service::ChainSpec>) {
```

Comment:
```bash
error: you seem to be trying to use `&Box<T>`. Consider using just `&T`
   --> runtime/mozart/cli/src/command.rs:131:36
    |
131 | fn set_default_ss58_version(_spec: &Box<dyn polkadot_service::ChainSpec>) {
    |                                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ help: try: `&dyn polkadot_service::ChainSpec`
    |
    = help: for further information visit https://rust-lang.github.io/rust-clippy/master/index.html#borrowed_box
    = note: `-D clippy::borrowed-box` implied by `-D clippy::complexity`
    = help: to override `-D clippy::complexity` add `#[allow(clippy::borrowed_box)]`
```

@tmpolaczyk I tried applying the recommended clippy fix but it results in:

```bash
error[E0277]: the trait bound `std::boxed::Box<dyn sc_cli::ChainSpec>: sc_cli::ChainSpec` is not satisfied
   --> runtime/mozart/cli/src/command.rs:173:30
    |
173 |     set_default_ss58_version(chain_spec);
    |                              ^^^^^^^^^^ the trait `sc_cli::ChainSpec` is not implemented for `std::boxed::Box<dyn sc_cli::ChainSpec>`
    |
    = help: the trait `sc_cli::ChainSpec` is implemented for `sc_service::GenericChainSpec<G, E, EHF>`
    = note: required for the cast from `&std::boxed::Box<dyn sc_cli::ChainSpec>` to `&dyn sc_cli::ChainSpec`

error[E0277]: the trait bound `std::boxed::Box<dyn sc_cli::ChainSpec>: sc_cli::ChainSpec` is not satisfied
   --> runtime/mozart/cli/src/command.rs:284:38
    |
284 |             set_default_ss58_version(chain_spec);
    |                                      ^^^^^^^^^^ the trait `sc_cli::ChainSpec` is not implemented for `std::boxed::Box<dyn sc_cli::ChainSpec>`
    |
    = help: the trait `sc_cli::ChainSpec` is implemented for `sc_service::GenericChainSpec<G, E, EHF>`
    = note: required for the cast from `&std::boxed::Box<dyn sc_cli::ChainSpec>` to `&dyn sc_cli::ChainSpec`

    ...
```

So I've added `#[allow(clippy::borrowed_box)]` for now.